src/ansys/fluent/core/solver/flobject.py

```diff
@@ -1799,7 +1784,7 @@ class Query(Action):
 
     def __call__(self, **kwds):
         """Call a query with the specified keyword arguments."""
-        newkwds = _get_new_keywords(self, [], kwds)
+        newkwds = _get_new_keywords(self, **kwds)
```

Comment:
The only difference between `Command` and `CommandWithPositionalArgs` is the call signature.